src/Rx.KitchenSink.ts

```diff
@@ -13,6 +13,8 @@ interface KitchenSinkOperators<T> extends CoreOperators<T> {
   min?: <T, R>(comparer?: (x: R, y: T) => R) => Observable<R>;
   timeInterval?: <T>(scheduler?: IScheduler) => Observable<T>;
   mergeScan?: <T, R>(project: (acc: R, x: T) => Observable<R>, seed: R) => Observable<R>;
+  switchFirst?: <R>() => Observable<R>
```

Comment:
Is type signature for `switchFirst` intended to be as below?
```
Observable<T> {
 switchFirst<R>:Observable<R> {
 }
}
```

In `switchFirst.ts`, it seems as
```
Observable<T> {
 switchFirst(): Observable<T> {
 }
}
```